src/m365/aad/commands/administrativeunit/administrativeunit-add.ts

```diff
@@ -0,0 +1,82 @@
+import { AdministrativeUnit } from "@microsoft/microsoft-graph-types";
+import GlobalOptions from "../../../../GlobalOptions.js";
+import { Logger } from "../../../../cli/Logger.js";
+import request, { CliRequestOptions } from "../../../../request.js";
+import GraphCommand from "../../../base/GraphCommand.js";
+import commands from "../../commands.js";
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  displayName: string;
+  description?: string;
+  hiddenMembership: boolean;
```

Comment:
This boolean option is actually `true` or `undefined` at runtime. So let's make it optional.

Reply:
fixed